src/main/java/com/revature/domain/EvalComment.java

```diff
@@ -79,21 +91,19 @@ public boolean equals(Object obj) {
 				return false;
 		} else if (!commentText.equals(other.commentText))
 			return false;
-		if (eval == null) {
-			if (other.eval != null)
-				return false;
-		} else if (!eval.equals(other.eval))
-			return false;
 		if (id == null) {
 			if (other.id != null)
 				return false;
 		} else if (!id.equals(other.id))
 			return false;
+		if (isActive != other.isActive)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1126)